.github/workflows/tests.yml

```diff
@@ -9,6 +9,7 @@ on:
   pull_request:
   schedule:
     - cron: '48 6 * * *'
+    - cron: '19 18 * * 0'
```

Comment:
If we do this we should change the postgis version of Python to 3.8, as explained in the floating commit to check the minimum version so far: the shapely version 1.7.0 specified as minimum requirements has only binary packages up to 3.8. Other Python versions require Shapely to be installed from source, but this requires the libgeos-dev library installed.